### DISCUSSION TITLE
Add `MIN SELECT` option to  `Relation` and `Select` fields

### DISCUSSION THREAD

**Discussion by babakfp** [pocketbase/pocketbase#1758] [Q&A] [answered]
Hey
Is there a reason that there is no such option for those field types?
Thanks

**Comment by ganigeorgiev** [comment, marked as answer]
I've added it in my todo to consider it since I'm planning some other minor changes to the multi-value fields as part of the next release.

As a workaround, you can define a rule constraint using the `:length` modifier, something like:

```js
yourMultiValueField:length > 3
```
